content/lessons/chapter-4/address-3.tsx

```diff
@@ -30,7 +30,9 @@ export default function Address3({ lang }) {
   }, [])
 
   const javascript = {
-    program: `console.log("KILL")`,
+    program: `
```

Comment:
We need to update the solution in `address.tsx` in resources to reflect this change.